Documentation/openssl.md

```diff
@@ -6,11 +6,13 @@ This is provided as a proof-of-concept guide to get started with Kubernetes clie
 
 ## Deployment Options
 
-The following variables will be used throughout this guide. The default for `K8S_SERVICE_IP` can safely be used, however `MASTER_IP` will need to be customized to your infrastructure.
+The following variables will be used throughout this guide. The default for `K8S_SERVICE_IP` can safely be used, however `MASTER_HOST` will need to be customized to your infrastructure.
 
-**MASTER_IP**=_no default_
+**MASTER_HOST**=_no default_
 
-The IP address of the master node. Worker nodes must be able to reach the master via this IP on port 443. Additionally, external clients (such as an administrator using `kubectl`) will also need access, since this will run the Kubernetes API endpoint.
+The address of the master node. In most cases this will be the publically routable IP of the master node. Worker nodes must be able to reach the master via this address on port 443. Additionally, external clients (such as an administrator using `kubectl`) will also need access, since this will run the Kubernetes API endpoint.
```

Comment:
instead of `the publically routable IP of the master node`, how about `a publicly routable IP or hostname of the master cluster`?